docling-service/main.py

```
@@ -0,0 +1,111 @@
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("docling-service")
+
+app = FastAPI(title="Ninja Docling Service", version="1.0.0")
+
+# Initialise converter once at startup — model loading is expensive
+try:
+    from docling.document_converter import DocumentConverter
+    converter = DocumentConverter()
+    logger.info("Docling DocumentConverter initialised successfully")
+except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)